.rubocop.yml

```diff
@@ -0,0 +1,25 @@
+AllCops:
+  Include:
+  - app/**/*.rb
+  - Rakefile
+  - '**/Gemfile'
+  - config.ru
+  Exclude:
+  - config/**/*
+  - db/**/*
+  - spec/**/*
+  - bin/**/*
+  - lib/tasks/*
+  TargetRubyVersion: 2.3
```

Comment:
Maybe update this ... it may reduce the number of autocorrects.

Reply:
Fixed, but sadly no, still 338 offenses. We could increase the maxes a bit more, and catch a few more of the method length issues, for example